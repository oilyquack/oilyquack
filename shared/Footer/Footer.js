import React from "react";

import { footer } from "./Footer.module.css";

const Footer = () => (
  <footer className={footer}>
    <span>&copy; Ollie Crook {new Date().getFullYear()}</span>
  </footer>
);

export default Footer;
