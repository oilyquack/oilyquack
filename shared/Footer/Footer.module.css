.footer {
  background: #fff;
  display: flex;
  justify-content: space-between;
  padding: 30px 5vw 30px 5vw;
  width: 100%;
}
