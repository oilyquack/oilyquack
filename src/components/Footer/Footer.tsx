import type { Component } from "solid-js";

import styles from "./Footer.module.css";

const Footer: Component = () => (
  <footer class={styles.footer}>
    <span>&copy; Ollie Crook {new Date().getFullYear()}</span>
  </footer>
);

export default Footer;
