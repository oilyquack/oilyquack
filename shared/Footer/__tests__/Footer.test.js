import { render } from "@testing-library/react";
import React from "react";

import Footer from "../Footer";

describe("shared : Footer", () => {
  it("renders correctly", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
