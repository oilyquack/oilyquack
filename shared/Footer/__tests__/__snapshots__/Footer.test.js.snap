// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shared : Footer renders correctly 1`] = `
<div>
  <footer
    class="footer"
  >
    <span>
      Â© Ollie Crook 
      2020
    </span>
  </footer>
</div>
`;
